openedx/core/djangoapps/schedules/models.py

```diff
@@ -6,6 +6,13 @@
 from config_models.models import ConfigurationModel
 
 
+EXPERIENCE_TYPES = (
+    (0, 'Recurring Nudge and Upgrade Reminder'),
+    (1, 'Course Updates'),
+)
```

Comment:
Can you change this to a `dict` so users of this don't need to know the positional values of each of the experiences?  For `choices` below you can pass in dict.values() and in resolvers.py you can use the dict's keys.